src/ServiceProvider.php

```diff
@@ -3,6 +3,7 @@
 namespace Statamic\Eloquent;
 
 use Illuminate\Foundation\Console\AboutCommand;
+use JetBrains\PhpStorm\Deprecated;
```

Comment:
This seems like a mistake. (You started using PhpStorm? 😄 )

Reply:
definitely a mistake.

i told you i'd give it a go... i don't hate it as much as i thought i would :)

Reply:
i think it added it when i added `@deprecated` in a comment